src/index.js

```diff
@@ -201,6 +201,7 @@ async function run(request, ctx) {
       site,
       unspreadLists: !!ctx.data.features?.unspreadLists,
       externalImageUrlPrefixes: ctx.data.features?.externalImageUrlPrefixes,
+      maxImages: ctx.data.limits?.maxImages,
```

Comment:
I would set the default here.... in case it is not passed along.

Reply:
Can do, thought about that but opted to follow the same way we handle the default for `externalImageUrlPrefixes`.

https://github.com/adobe/helix-html2md/pull/772/files#diff-fb5cbd5a02083f1a9d0ecf784cb80d09e8eb5eaf726ef4878ab0464e517bd5e9R31